### DISCUSSION TITLE
GTP 3 Integration ?

### DISCUSSION THREAD

**Discussion by rayenCtrl** [brianpetro/obsidian-smart-connections#284] [Ideas]
Is it possible to add GPT 3

**Comment by brianpetro** [comment]
Hi @rayenCtrl 

Can you elaborate?

Smart Chat already integrates with multiple GPT chat models through OpenAI.

Thanks
🌴

**Reply by rayenCtrl** [reply]
I don't have an 3.5 or gpt 4 account so when I put my key in the smart connection it doesn't work, is there a possibility for a free version

**Reply by rayenCtrl** [reply]
@brianpetro Did you get it ?

**Reply by skchaudr** [reply]
If you have an API key, that means you have an OpenAI account, and either have free credits ($18 worth), or have a payment method added. 

If you do not have an account at all, you can't sign in and use the free version either. 

The OpenAI API doesn't require a Premium or Plus Subscription to ChatGPT.

**Reply by rayenCtrl** [reply]
I know that @skchaudr but my question was my account is free account (not 3.5 or 4 version) and no plugin work for me it all need a 3.5 or 4 version plugin so why it cant work on the default version gpt 3

**Reply by JasonBates** [reply]
You can use ChatGPT through the openAI website for free, but if you are using another app or plugin to talk to chatGPT behind the scenes (via an API), there is no free version of that. 

OpenAI give you some free credits to use at the start to try it, but then you have to pay for API access, which can come out to a few dollars a day for me personally. 

Unfortunately there is no free version that would work with Smart Connections, even though you can use the free ChatGPT website,

**Reply by rayenCtrl** [reply]
@JasonBates Thank you for the explanation that was my question
